payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -451,7 +451,7 @@ class Stripe internal constructor(
         clientSecret: String,
         stripeAccountId: String? = this.stripeAccountId,
         expand: List<String> = emptyList(),
-    ): PaymentIntent? {
```

Comment:
Tightening the return type as proposed [here](https://github.com/stripe/stripe-android/pull/6561#discussion_r1171733319).